input/data/2008/2008ApJ...679.1427A/tev-000014-sed.ecsv

```diff
@@ -0,0 +1,24 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
```

Comment:
So far, whenever we have a published differential spectrum, i.e. `dnde`, the energy `e_ref` at which the diff flux is `dnde` was given. Is this not the case here?